app/src/main/java/com/macbitsgoa/events/speakers/SpeakerCardFragment.java

```diff
@@ -0,0 +1,50 @@
+package com.macbitsgoa.events.speakers;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+
+import com.google.android.material.card.MaterialCardView;
+import com.macbitsgoa.events.R;
+import com.macbitsgoa.events.home.HomeCardInterface;
+
+import androidx.fragment.app.Fragment;
+
+/*
+ * Created by Hariram.R
+ * 13/06/2018
+ * */
+
+public class SpeakerCardFragment extends Fragment implements View.OnClickListener,
+        HomeCardInterface {
+
+    private MaterialCardView cardView;
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        cardView = (MaterialCardView) inflater.inflate(R.layout.fragment_speakers_card,
+                container, false);
+        return cardView;
+    }
+
+    @Override
+    public void onClick(View view) {
+        Intent intentToSpeakersActivity = new Intent(view.getContext(), SpeakersActivity.class);
```

Comment:
[just suggestion] action + type (= `lauchSpeakersIntent`) would've been better choice for var name